package/linux-firmware/Config.in

```diff
@@ -216,6 +216,42 @@ config BR2_PACKAGE_LINUX_FIRMWARE_CYPRESS_CYW54XXX
 	help
 	  Firmware files for the Cypress cyw54xxx FullMAC.
 
+config BR2_PACKAGE_LINUX_FIRMWARE_IWLWIFI_100
+	bool "Intel iwlwifi 100"
+	help
+	  Firmware files for the Intel Wifi 100 devices supported by
+	  the iwlwifi kernel driver.
+
+config BR2_PACKAGE_LINUX_FIRMWARE_IWLWIFI_105
+	bool "Intel iwlwifi 105"
+	help
+	  Firmware files for the Intel Wifi 105 devices supported by
+	  the iwlwifi kernel driver.
+
+config BR2_PACKAGE_LINUX_FIRMWARE_IWLWIFI_135
+	bool "Intel iwlwifi 135"
+	help
+	  Firmware files for the Intel Wifi 135 devices supported by
+	  the iwlwifi kernel driver.
+
+config BR2_PACKAGE_LINUX_FIRMWARE_IWLWIFI_1000
+	bool "Intel iwlwifi 135"
```

Comment:
```suggestion
	bool "Intel iwlwifi 1000"
```

Reply:
D'oh, I checked it like three times, yet one still slipped to me :see_no_evil: Upstream patch updated, downstream force-pushed. Thanks!